OpenMEEG/src/TestHead.txt

```diff
@@ -26,6 +26,7 @@ function(TESTHEAD HEADNUM)
     set(DIPPOS                 ${MODELBASE}.dip)
     set(DIPPOS-SKULLSCALP      ${MODELBASE}-skullscalp.dip)
     set(PATCHES                ${MODELBASE}.patches)
+    set(EITPATCHES             ${MODELBASE}.EITpatches)
```

Comment:
I would rename to : ${MODELBASE}-EIT.patches
unless .patches is different as the file format is different.